tests/Renderer/TaskLineRenderer.test.Visualise_HTML_Minimal_task_-_full_mode.approved.html

```diff
@@ -9,7 +9,7 @@
   data-line="0"
   data-task-status-name="Cancelled"
   data-task-status-type="CANCELLED">
-  <input class="task-list-item-checkbox" type="checkbox" title="Right-click for options" data-line="0" />
```

Comment:
The so-called minimal test task used to generate this output does not have a path, hence this change.